brvs-core/src/main/resources/application.properties

```diff
@@ -39,3 +39,4 @@ BILLING_EXCHANGERACCOUNT=exchanger_fiat@notary
 
 MIN_SIGNATORIES_RULE_AMOUNT=3
 QUORUM_DIVISOR_RULE_VALUE=2
+BLOCKED_ASSETS=blocked#sora
```

Comment:
Is  this just a fake asset to allow create a bean?

Reply:
Yes